src/Select/SelectInput.js

```diff
@@ -413,6 +419,10 @@ SelectInput.propTypes = {
    * You can only use it when the `native` property is `false` (default).
    */
   renderValue: PropTypes.func,
+  /**
+   * Tab index of the component
+   */
```

Comment:
This prop can probably be `@ignore`ed, since it's a standard DOM attribute (albeit being applied to a nested div), and since this is an internal component.